pandas/core/frame.py

```diff
@@ -2218,7 +2219,7 @@ def query(self, expr, inplace=False, **kwargs):
         else:
             return new_data
 
-    def eval(self, expr, inplace=None, **kwargs):
+    def eval(self, expr, inplace=True, **kwargs):
```

Comment:
missed this one when reverting all of my original changes, thanks for pointing out!